src/main/java/net/mcreator/ui/action/accelerators/ActionAccelerator.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2022, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.action.accelerators;
+
+import javax.swing.*;
+import java.awt.event.KeyEvent;
+
+/**
+ * <p>This class is used by the {@link net.mcreator.ui.action.BasicAction} accelerators.</p>
+ */
+public class ActionAccelerator extends Accelerator {

```

Comment:
In this PR, it seems useless, but I made this class because when this PR will be merged, I planned to also allow changing the mod element shortcuts (when creating one). However, this will require to use of accelerators and to differentiate them, I started to use a word at the beginning such as the `action.`. If I move those constructors inside the main class, we will have to write everytime `action.` or `modelement.` (for example). It was only to make things a little bit easier. For the future, it might also help to already have different accelerator types if we want to add a feature for a specific type. It will make the job easier.

Reply:
Technically, these constructors can be moved to the super class as there would be no signature conflicts.

Reply:
As I said, it would work for this PR, but Java is not smart enough yet to differentiate constructors that have the same arguments, but don't do the same thing (which will be the case with custom accelerators for mod elements)